platform/kobo/koreader.sh

```diff
@@ -133,8 +133,12 @@ if [ -z "${ksmroot+x:?}" ]; then
     cat "/sys/class/graphics/fb0/rotate" >"/sys/class/graphics/fb0/rotate"
 fi
 
-./reader.lua "${args}" >>crash.log 2>&1
-RESULT=$?
+RESULT=85
```

Comment:
Is there a reason that in some scripts it's `RESULT` and in others `RETURN_VALUE`?